packages/react/src/components/Table/TableManageViewsModal/TableManageViewsModalContent.mdx

```diff
@@ -0,0 +1,147 @@
+| actions.onClearError            | function                        |                                                     | Called when the dialog error msg is cleared                                                 |
+| actions.onSearchChange          | function                        |                                                     | Called when the search input value is changed, the search term is passed as argument.       |
+| actions.onDisplayPublicChange   | function                        |                                                     | Called when view title input value is changed, a bool value is passed as argument.          |
+| defaultViewId                   | string                          | undefined                                           | The ID of the view that should be marked as default if any                                  |
+| displayPublicDefaultChecked     | bool                            | true                                                | True means that the 'display public' checkbox is checked by default                         |
+| error                           | string                          | undefined                                           | Shows this string as a general modal error when present                                     |
+| isLoading                       | bool                            | false                                               | Shows loading skeleton and disables pagination when true                                    |
+| searchValueDefault              | string                          | ''                                                  | The default value of the search field value                                                 |
+| open                            | bool                            |                                                     | Determines if the modal is open or closed (i.e. visible or not to the user)                 |
+| overrides                       | shape                           | undefined                                           | Used to override the internal components and props for advanced customisation               |
+| pagination                      | shape                           | undefined                                           | pagination at the bottom of list, see SimplePaginationPropTypes                             |
+| testId                          | string                          | 'TableManageViewsModal'                             | Id that can be used for testing                                                             |
+| views                           | [ViewsPropType](#viewsproptype) |                                                     | The views to be displayed in the list, see ViewsPropType                                    |
```

Comment:
The link for ViewsPropType does not work.

Reply:
Good catch, fixed